dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -20,11 +20,20 @@ void NotificationsLongRunningPlatformImpl::Initialize()
     // This is in case we later realize there are no apps to be tracked in the LRP.
     m_lifetimeManager.Setup();
 
-    m_storage = Storage::ApplicationData::Current().LocalSettings().CreateContainer(
-        L"LRP", Storage::ApplicationDataCreateDisposition::Always);
+    m_rawStorage = Storage::ApplicationData::Current().LocalSettings().CreateContainer(
+        L"Raw", Storage::ApplicationDataCreateDisposition::Always);
+
+    m_toastStorage = Storage::ApplicationData::Current().LocalSettings().CreateContainer(
```

Comment:
Can the toast storage be managed by the ToastRegistrationManager? It seems odd to me that the actual registration process happens partially in the component, but this step doesn't.

Reply:
This is a great idea. I'll move the storage to the ToastRegistrationManager class.